uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/leave-request-instance.js

```diff
@@ -120,18 +144,12 @@ define([
          * @return {object}
          */
         defaultCustomData: function () {
+          loadSupportedFileTypes.call(this);
```

Comment:
Yes it was possible in case the API throws an error. Changed the logic a bit, now "this.fileUploader" is set in the `finally`.